cloud-operator/templates/role.yaml

```
@@ -21,4 +21,15 @@ rules:
 - apiGroups: [""]
   resources: ["secrets"]
   resourceNames: ["hubble-relay-client-certs"] # Name of the Secret
-  verbs: ["get"]
\ No newline at end of file
+  verbs: ["get"]
+---
+# Role to read secrets in the GKE managed observability namespace
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: illumio-cloud-operator-secret-reader
+  namespace: gke-managed-dpv2-observability
+rules:
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["get", "list"]
```

Why would "list" be required for namespace `gke-managed-dpv2-observability` and not for `kube-system`, even though the logic to read the secret is identical for either namespace?
```suggestion
  verbs: ["get"]
```